Mesh_3/include/CGAL/Labeled_mesh_domain_3.h

```diff
@@ -632,10 +632,10 @@ class Labeled_mesh_domain_3
    *   \cgalParamNBegin{features_detector}
    *    \cgalParamDescription{ a functor that implements
    *      `std::vector<std::vector<Point>> operator()(const Image_3& img) const`,
-   *      where `%Point` matches the mesh domain point type,
-   *      that returns a range of detected polyline features for feature protection.
-   *      Polyline features are added to the domain for further feature protection.
-   *      See \ref PkgMesh3FeatureDetection for available functors.}
+   *      where `%Point` matches the mesh domain point type.
+   *      It returns a range of detected polyline features, which can be added
```

Comment:
actually the features **are** added inside `create_labeled_image_mesh_domain`. This version seems to suggest that the user can add them later, don't you think?

Reply:
```suggestion
   *      It returns a range of detected polyline features, which are added
```